TAC/Meetings/2023/2023-03-09/CCC-TAC-Minutes-2023-03-09.md

```diff
@@ -0,0 +1,168 @@
+* Howard Huang will come up with a project progression state draft by the end of next week (3/17)
+* AF to sync with Dave Thaler for the blog post on attestation addition
+* Gramine website refresh reboot is making progress, brief being updated to see if there was a way to come back with a lower cost
+* DM/KT need to sync on a CCC security mailing list, discuss those to be added to the list
+* Need update from Lily on Keystone questions
+* KT commented that the needs to find access to the Keystone website to fix the programs link
+
+
+**2023 Goals**
+
+* DM discussed 2023 plans
+  * Cross-org/Cross SIG 
+    * Regulatory work (GRC SIG)
+      * MN gave status on objectives that will be updated in April review
+    * FHE educational content
+  * Community development
+    * Outreachy update next meeting
+
+
+**Project Progression**
+
+* DM overviewed the [progression policy](https://github.com/confidential-computing/governance/blob/main/project-progression-policy.md)
+* Gramine has requested to progress in state
+* Eric Voit discussed the origin (CNCF) and that it has a different focus than the CCC
+  * What are the additional services do projects need or want per stage of progression
+  * Only benefit now is visibility (publicity) on the website, no per-state additional benefits at present, though there could be.
+  * [Benefits list](https://github.com/confidential-computing/governance/blob/main/project-progression-policy.md#benefits-of-being-a-recognized-consortium-project)
+* Only approved stages are sandbox and incubation, the rest of the document is a proposal and has not been approved
+* DT discussed if there was a real need to have different stages
+* Mentors to get input from their projects for discussion in the next meeting
+
+
+**TAC Tech Talk**
+
+* eBPF - Dave Thaler
+  * DT presented an overview of eBPF and how it relates to Confidential Computing
+
+
+**Outreach Update**
+
+* DM summarized OC3
+* Helen Lau reported that Ben Fischer is delivering the Keynote to OC3 instead of Stephen Walli
+* Mark Novak talked about his presentation - on what stands in the way of broad adoption
+* DM talked about a few other talks
+
+
+**Conformance to Terminology**
+
+* Alec Fernandez discussed the [NVIDIA blog](https://blogs.nvidia.com/blog/2023/03/01/what-is-confidential-computing/)
+  * The diagram that shows TEE and the AMD SEV-ES environment, and other technologies like Homomorphic Encryption under the heading of Confidential Computing - does this meet the definition of Confidential Computing?
+  * DM noted that the diagram may have been created before CCC Terminology was published
+  * Alec concerned with referring to CC in a blog post with an opinion that is not broadly agreed to as being correct
+  * DM posed a GB question for conformance enforcement, DT volunteered Mike Bursell to send email to NVIDIA for possible cleanup of the blog, AF to reach out to MB
```

Comment:
"AF to reach out to MB" sounds like an action item, but it's not marked with **ACTION**